1-exercises/G-array-methods/exercise.js

```diff
@@ -4,7 +4,7 @@
 */
 
 let numbers = [3, 2, 1];
-let sortedNumbers; // complete this statement
+let sortedNumbers = numbers.sort(); // complete this statement
```

Comment:
be reminded that default sort works by storing elements as String in alphabetic order. Better sort with a numeric sorting function.